windows-nsis/openvpn.nsi

```diff
@@ -228,8 +231,15 @@ Section /o "-workaround" SecAddShortcutsWorkaround
 	; as we don't want to move SecAddShortcuts to top of selection
 SectionEnd
 
-Section /o "${PACKAGE_NAME} User-Space Components" SecOpenVPNUserSpace
+Section /o "-launchondummy" SecLaunchGUIOnLogon0
```

Comment:
Both _SecAddShortcuts_ and _SecLaunchGUIOnLogon_ are in a separate SectionGroup. I'm wondering whether we truly need this trick or not. Any ideas?